### PR TITLE
Fix #50, Updates Guide-GroundSystem.txt and CheaderParser.py from pyt…

### LINKED ISSUE #50: Update build/run instructions for python 3
**Is your feature request related to a problem? Please describe.**
Guide-GroundSystem.txt is now out of date based on python 3 updates.

**Describe the solution you'd like**
Update the install/run guide

**Describe alternatives you've considered**
Move to a wiki, document in README, etc.

**Requester Info**
Jacob Hageman - NASA/GSFC

### PR DESCRIPTION
**Pull request by dmknutsen**
**Describe the contribution**
Fix #50, Updates Guide-GroundSystem.txt and CheaderParser.py from python 2-->3

**Testing performed**
Steps taken to test the contribution:
1. Built new VM (Ubuntu OS) and verified that updated installation instructions work as expected.
2. Used CHeaderParser.py to add commands for a custom cFS application and verified that associated commands are able to get sent successfully.

**Expected behavior changes**
Guide-GroundSystem.txt and CheaderParser.py updated from python 2-->3

**System(s) tested on**
 - Oracle VM VirtualBox
 - OS: ubuntu-19.10
 - Versions: cFE 6.7.6.0, OSAL 5.0.6.0, PSP 1.4.4.0

Contributor Info
Dan Knutsen
GSFC/NASA

### CODE REVIEW

**Comment by mprather** [comment]
From a clean install on Ubuntu 18.04, I need 1 additional package to avoid any warnings during startup of the GroundSystem script, the libcanberra-gtk-module. Without this library, the script always reports Gtk-Message: <time>: Failed to load module "canberra-gtk-module".

**Comment by astrogeco** [comment]
CCB 20200318 - Add an "outdated" notice for CentOS and Mac instructions instead of removing them.

File: Guide-GroundSystem.txt [hunk 1]
```diff
@@ -23,59 +23,17 @@ Before launching the Ground System make sure that:
...
-    $ sudo apt-get install python-zmq
+    $ sudo apt-get install python3-pyqt4
+    $ sudo apt-get install python3-zmq
+    $ sudo apt-get install libcanberra-gtk-module
     $ ( cd Subsystems/cmdUtil/ && make )
-    $ python GroundSystem.py
+    $ python3 GroundSystem.py
 
 
-Installing and running cFS Ground System on CentOS 6:
```
Comment (astrogeco) [inline Guide-GroundSystem.txt]: Add these back in and note that they're outdated.

Comment (dmknutsen) [reply Guide-GroundSystem.txt]: I made the changes yesterday.  We should be good to go.

**Pull request merged**